packages/app/src/components/promo/PromoAction.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    data-cy="promo-action"
+    class="flex flex-row items-center gap-[12px] h-[32px] bg-gray-50 border border-gray-100 rounded w-fit text-sm"
+  >
+    <div class="rounded-l border-r-gray-100 flex flex-row ml-[12px] items-center justify-center h-full">
+      <component
+        :is="leftIcon"
+        v-if="leftIcon"
+        class="icon-dark-gray-500 icon-light-gray-200"
+      />
+      <span class="text-purple-500 whitespace-nowrap">
+        {{ leftLabel }}
+      </span>
+    </div>
+
+    <div class="bg-white h-full w-full">
+      <ExternalLink
+        v-if="href"
+        data-cy="promo-action-link"
+        :href="href"
+        class="rounded-r bg-white items-center flex flex-row text-indigo-500 h-full"
+      >
+        <span
+          class="flex flex-row items-center font-medium mx-[12px] whitespace-nowrap"
+        >
+          {{ rightLabel }}
+          <component
+            :is="rightIcon"
+            v-if="rightIcon"
+            class="ml-[8px]"
+          />
+        </span>
+      </ExternalLink>
```

Comment:
Is this `ExternalLink` part ever used? I only see the `href` prop being used in a test.

Reply:
I had originally thought the "Tour" button was a link out to the docs site so I built that version of the button, but I never circled back to remove it. It's valid and tested, but currently dead code. Ever envision us needing this? If not I'll go ahead and remove

Reply:
I don't see a reason at this time to keep it. There was a variant of the designs at one point that did have that as a link out, but it was scratched.